src/lib/piwik.js

```diff
@@ -0,0 +1,126 @@
+/* global __PIWIK_TRACKER_URL__ __PIWIK_SITEID__ __PIWIK_DIMENSION_ID_APP__ */
```

Comment:
I'm guessing there's no wya to import this from cozy-client-js to avoid duplication? In that case did you consider removing the unused parts, like the middleware for example?